### PR TITLE
Add controls to generate code snippet

### LINKED ISSUE #27: Embeddable code snippets
**Is your feature request related to a problem? Please describe.**

As a playground user, I would like a way to create embeddable snippets from a playground project. I would like to embed these snippets in blog posts, tutorials and documentation.

**Describe the solution you'd like**

It would be nice if I could create a snippet from any pane in the playground -- a specific account, transaction or script. This snippet could then be embedded as HTML on any webpage. The embedded snippet should display Cadence syntax highlighting. It should also include a link to the original playground project.

**Describe alternatives you've considered**

It's currently possible to create a GitHub Gist that contains Cadence code, but it does not support syntax highlighting and is not runnable in the browser.

### PR DESCRIPTION
**Pull request by MaxStalker**
Closes: #27 

## Description
Add button to sidebar menu items to copy snippet of JavaScript code, which will allow embedding of Cadence code snippet into docs and blogs.

Possible enhancements:
- create separate button and popup to allow customization (theme selection) and maybe preview of the output 🤔

### CODE REVIEW

**Comment by MaxStalker** [comment]
To the reviewer, please ignore hardcoded API and projectId 😅

File: src/api/apollo/client.ts [hunk 1]
```diff
@@ -7,15 +7,15 @@ import localResolvers from "./resolvers";
 
 import { ResultType } from "./generated/graphql";
 
-const api = process.env.GRAPHQL_API;
+const api = process.env.PLAYGROUND_API
```
Comment (turbolent) [inline src/api/apollo/client.ts:10]: Can you please also open a PR to update the deployments:

- https://github.com/dapperlabs/flow-playground/blob/25e0593b6ee4c012aa3b2e4642bb7ef92cb05d10/Makefile#L20
- https://github.com/dapperlabs/flow-playground/blob/25e0593b6ee4c012aa3b2e4642bb7ef92cb05d10/Makefile#L34

File: src/components/AccountList.tsx [hunk 2]
```diff
@@ -37,18 +39,26 @@ const AccountList: React.FC = () => {
...
+              title={title}
+              active={isActive}
               onClick={() => setActive(EntityType.Account, i)}
             >
               <AccountCard>
                 <Avatar seed={project.seed} index={i} />
                 <Stack>
                   <strong>0x{account.address.slice(-2)}</strong>
-                  <small>{getDeployedContracts(account) || "--"}</small>
+                  <small>{contractName}</small>
```
Comment (turbolent) [inline src/components/AccountList.tsx:58]:
```suggestion
                  <small>{contractName || "--"}</small>
```

File: src/components/AccountList.tsx [hunk 3]
```diff
@@ -37,18 +39,26 @@ const AccountList: React.FC = () => {
       <Header>Accounts</Header>
       <Items>
         {project.accounts.map((account: Account, i: number) => {
+          const isActive = active == i
+          const contractName = getDeployedContracts(account) || "--"
+          const title = contractName !== "--"
+            ? `${contractName} is deployed to this account`
+            : `This account don't have any contracts`
```
Comment (turbolent) [inline src/components/AccountList.tsx:46]:
```suggestion
          const contractName = getDeployedContracts(account)
          const title = contractName ?
            ? `${contractName} is deployed to this account`
            : `This account don't have any contracts`
```

File: webpack.config.js [hunk 4]
```diff
@@ -47,6 +48,7 @@ module.exports = {
   devServer: {
     contentBase: path.join(__dirname, 'dist'),
     disableHostCheck: true,
+    historyApiFallback: true,
```
Comment (turbolent) [inline webpack.config.js:51]: What does this do and why is this needed?

**Review by turbolent — APPROVED** [review]
Nice, looks good @MaxStalker!

Just a minor suggestion and a question

File: webpack.config.js (hunk 4 above)
Comment (MaxStalker) [reply webpack.config.js:51]: If I got it right, on devserver if page is not found it shall it redirected to `index.html` instead of showing 404.
I had some issues, when saving the project and page would show me 404 instead of actual project with updated url. 
Those updates fixed the issue.

File: src/api/apollo/client.ts (hunk 1 above)
Comment (MaxStalker) [reply src/api/apollo/client.ts:10]: Done :)
https://github.com/dapperlabs/flow-playground/pull/121

File: src/components/AccountList.tsx (hunk 2 above)
Comment (MaxStalker) [reply src/components/AccountList.tsx:58]: This computation is slightly above, where we defined `contractName` constant, so we don't need to make it "inline" 😅

Comment (MaxStalker) [reply src/components/AccountList.tsx:58]: Nvm, I see what you suggest  👍

File: webpack.config.js (hunk 4 above)
Comment (10thfloor) [reply webpack.config.js:51]: Basically, always serve "index.html", regardless of the URL. React Router decides what to show based on the URL, not the webserver.

**Review by 10thfloor — APPROVED** [review]

**Review by 10thfloor — APPROVED** [review]
